### ISSUE TITLE
Order Preview > Order Tabs - Order Details 

### ISSUE THREAD

**Issue by boxxroom** [damanic/Lemonstand-V1__Installer#1]
First of all thank you for keeping with updates for LS stores, it helps quite a bit, especially with the dashboard report generation.

Today, using all of the available modules selected, we've updated some of our stores and have noticed that for some odd reason the Totals (Total, Subtotal (Before Discounts), Tax Total etc..) are not being pulled through and displaying.

![totals](https://user-images.githubusercontent.com/33870545/36972878-bcce9d46-2068-11e8-989b-7465b4861c6e.png)

**Comment by damanic** [comment]
Is this from a fresh install using this installer, or an existing install you have updated with damanic/ls1-module-updatecenter ?

**Comment by damanic** [comment]
So the data is all there, but not being displayed in the preview fields only? Could this be a browser render issue. If you look at the HTML output are the values there?

![frompreview-issue](https://user-images.githubusercontent.com/5227303/37011449-5cb8cd32-2144-11e8-80e2-8d38a2a29b73.PNG)

**Comment by boxxroom** [comment]
No, these installs are approx 4-5 (if not more) years old with plenty of historic data.

I believed I have found the issue/culprit https://github.com/damanic/ls1-module-shop/commit/689a56d36b475e56e7e9100ddf65aa799aa6c155 which is stopping the rendering of orders totals which were placed before the update.

I have created PR to fix this, make it backwards compatible for historic orders and for any futures Lemonstand devs installing these updates.
